File: Programmers/Lv0/글자 지우기.py
"""
출처: 프로그래머스 코딩 테스트 연습,
https://school.programmers.co.kr/learn/courses/30/lessons/181900
"""
# 내 풀이
def solution(my_string, indices):
    result=[]
    for x in range(len(my_string)):
        if x in indices:
            continue
        else:result.append(my_string[x])
    answer = ''.join(result)
    return answer

# 다른 사람 풀이
def solution(my_string, indices):
    answer = ''
    for i in range(len(my_string)):
        if i not in indices:answer+=my_string[i]
    return answer

File: Programmer_Java/Lv0/글자 지우기.java
// package Programmers;

// public class 글자 지우기 {
  
// }

// 출처: 프로그래머스,
// https://school.programmers.co.kr/learn/courses/30/lessons/181900

// 풀이 과정
/*
 * class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = "";
        
        String[] split_string = my_string.split("");
        
        for(int i = 0; i<split_string.length; i++){
            Boolean flag = false;
            
            for(int k: indices){
                if(i==k){
                    flag =true;
                }
            }
            if(flag == false){
                answer+=split_string[i];
            }
            
        }
        
        
        return answer;
    
    
    }
}
 */